dog-button.js

```diff
@@ -0,0 +1,92 @@
+
+let total = 0
+
+function clickPhoto(name, breed, cost) {
+  let result = "Dog name: " + name + " Breed: " + breed + " Fees: " + cost
+  alert(result);
+}
+function dogFees(fee) {
+   total = total + fee
+}
+/*track cart total*/
+$('.button').click(function(){
+  $('#total').text(total);
+});
+
+/*hover effect on dog tiles*/
+$('.col-1').hover(function(e){
+  $(this).addClass('tile-border');
+})
+$('.col-1').mouseleave(function(e){
+  $(this).toggleClass('tile-border');
+})
+/*Handle form data on Checkout page*/
+$("form").submit(function(){
+  let text = $("#name").val()
```

Comment:
Very good! Only thing I would mention is your form is redirecting on submit. We don't want this to happen. 
use event.preventDefault() to keep this from happening. 

`$('form').submit(function (evt) {
   evt.preventDefault(); //prevents the default action

}`